### ISSUE TITLE
"Please file an issue: reasonType= interrupted"

### ISSUE THREAD

**Issue by darkwing** [devtools-html/debugger.html#4507]
While stepping through a breakpoint series, I saw this in the console:

```
TypeError: source is undefined[Learn More]  ast.js:129:31
Please file an issue: reasonType= interrupted pause.js:89:4
```

**Comment by codehag** [comment]
Is there any more info about this? ie an STR?

**Comment by jasonLaster** [comment]
closing as we're going to fix it here #4972
